morph/react/property-text.js

```diff
@@ -8,6 +8,23 @@ import {
 import safe from './safe.js'
 import wrap from './wrap.js'
 
+const parseFormatValue = (value, type) => {
+  if (type === 'percent') {
+    return value / 100
+  }
+  if (type === 'date') {
+    return `Date.parse('${value}')`
+  }
+  if (type === 'time') {
+    const timeValues = value.split(':')
+    let timeStr = `Date.UTC(2018, 14, 3`
```

Comment:
I figure the fixed values here don't matter, do them? or should we always pick up the current date for that?

Reply:
Yeah, the date doesn't matter. I mean I can pick up the current date if you want but it will have no impact to the output

Reply:
good stuff, let's leave it like that then :)